tests/sparsezoo/analysis/test_helpers.py

```diff
@@ -0,0 +1,368 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import onnx
+import pytest
+
+from sparsezoo import Zoo
+from sparsezoo.analysis import (
+    get_layer_and_op_counts,
+    get_layer_param,
+    get_node_four_block_sparsity,
+    get_node_four_block_sparsity_sizes,
+    get_node_sparsity,
+    get_node_sparsity_sizes,
+    get_zero_point,
+    is_four_block_sparse_layer,
+    is_parameterized_prunable_layer,
+    is_quantized_layer,
+    is_sparse_layer,
+)
+
+
+# Helper functions and dictionaries
+model_stubs = {
+    "yolact_none": "zoo:cv/segmentation/yolact-darknet53/"
+    "pytorch/dbolya/coco/base-none",
+    "mobilenet_v1_pruned_moderate": "zoo:cv/classification/mobilenet_v1-1.0/"
+    "pytorch/sparseml/imagenet/pruned-moderate",
+    "bert_pruned_quantized": "zoo:nlp/question_answering/bert-base/"
+    "pytorch/huggingface/squad/"
+    "12layer_pruned80_quant-none-vnni",
+}
+
+# TODO: There might be a better way to code this
+#       that doesn't require writing the onnx to disk
+model_onnxs = {}
+for model_name, stub in model_stubs.items():
```

Comment:
we should be able to refactor this as a parameterized pytest fixture - see here https://docs.pytest.org/en/6.2.x/fixture.html